rs/nns/governance/src/governance/tla/mod.rs

```diff
@@ -0,0 +1,179 @@
+use itertools::Itertools;
+use std::collections::BTreeMap;
+use std::thread;
+
+use super::Governance;
+use crate::storage::with_stable_neuron_indexes;
+
+pub use tla_instrumentation::{
+    Destination, GlobalState, InstrumentationState, Label, ResolvedStatePair,
+    TlaConstantAssignment, TlaValue, ToTla, Update, UpdateTrace, VarAssignment,
+};
+pub use tla_instrumentation_proc_macros::tla_update_method;
+
+pub use tla_instrumentation::checker::{check_tla_code_link, PredicateDescription};
+
+pub use ic_nervous_system_tla::{
+    account_to_tla, opt_subaccount_to_tla,
+    store::{TLA_INSTRUMENTATION_STATE, TLA_TRACES},
+    subaccount_to_tla,
+};
+pub use ic_nervous_system_tla::{tla_log_locals, tla_log_request, tla_log_response};
+
+use std::path::PathBuf;
+
+use icp_ledger::Subaccount;
+
+mod common;
+mod split_neuron;
+pub use split_neuron::split_neuron_desc;
+
+fn neuron_global(gov: &Governance) -> TlaValue {
+    let neuron_map: BTreeMap<u64, TlaValue> = gov
+        .neuron_store
+        .heap_neurons()
```

Comment:
We are in the process of removing "heap_neurons" and migrating to stable memory.

Do we need a global neuron state for TLA to work?

Reply:
Kind of - since the behavior of the methods (in the code) depends on the global neuron state, the most straightforward (and least error prone) way of modeling this is to reflect this global state. How would you recommend I go about collecting the neuron store (for test purposes)?

Reply:
Okay - this is currently just a subset of neurons, not including the "inactive" neurons.  It will not actually give you the global neuron state.

Reply:
What would you recommend using as a future proof solution, to obtain all the neurons? I saw that there was a `clone_neurons` method on `NeuronStore`, but that also uses `heap_neurons` and has a TODO pointing to a NNS-2474 issue which I couldn't find for some reason, so I wasn't sure if I should use that.

Reply:
Solution: use `with_stable_neuron_store` and then (on that) `range_neurons` with a full range to also get the stable neurons.

Reply:
Added now